modules/core/src/main/scala/fs2/kafka/security/TrustStoreFile.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018-2021 OVO Energy Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package fs2.kafka.security
+
+import cats.effect.Sync
+import cats.syntax.all._
+
+import java.nio.file.Path
+
+sealed abstract class TrustStoreFile {
+  def path: Path
+
+  def pathAsString: String
```

Comment:
I don't think we need these `pathAsString` methods - let's just do `path.toString` at the use site?

Reply:
Yeah I wondered about that too, will remove.